### PR TITLE
Top rated clubs

### PR DESCRIPTION
**Pull request by rugvedcbt**
Displayed top rated clubs instead of hard coded clubs in home page and added a constant for tenant in settings for defining specific tenant

### CODE REVIEW

File: backend/base/views.py [hunk 1]
```diff
@@ -51,7 +51,8 @@
 from dotenv import load_dotenv
 load_dotenv()
 DEBUG = os.environ.get('DJANGO_DEBUG')
-if DEBUG:
+if DEBUG == 'True':
+    print('local')
```
Comment (ramprasadm-CBT) [inline backend/base/views.py]: Please remove unwanted print comments

File: backend/base/views.py [hunk 2]
```diff
@@ -222,6 +223,7 @@ def post(self, request, *args, **kwargs):
                 request.session['first_login_' + str(user.id)] = True
 
             current_site = get_current_site(request)
+            print(current_site, 'is the current site')
```
Comment (ramprasadm-CBT) [inline backend/base/views.py]: Remove this please.

**Review by ramprasadm-CBT — APPROVED** [review]

File: backend/base/views.py (hunk 1 above)
Comment (rugvedcbt) [reply backend/base/views.py]: removed

File: backend/base/views.py (hunk 2 above)
Comment (rugvedcbt) [reply backend/base/views.py]: removed

**Pull request merged**